### PR TITLE
[ads] Disable CustomNotificationAdsStudy study

### PR DESCRIPTION
**Pull request by aseren**
The `CustomNotificationAdsStudy` study is disabled now to ensure push notification ads function consistently across Nightly, Beta, and Release channels (the study was enabled for Nightly and Beta only).

### CODE REVIEW

**Comment by btlechowski** [comment]
Verified with

Brave | 1.72.51 Chromium: 129.0.6668.70 (Official Build) nightly (64-bit)
-- | --
Revision | 06fad829365844d6f91d8f6f1fa1e87d1c9e611c
OS | Linux

Run Brave with `--variations-pr=1218`

Verified `CustomNotificationAdsStudy` is not listed

image


Verified System ads are shown:

image

----------------

Verified with

Brave | 1.71.105 Chromium: 129.0.6668.89 (Official Build) beta (64-bit)
-- | --
Revision | f69ba167992437799f1dbcce81e90ba501a3d63e
OS | Linux

Run Brave with `--variations-pr=1218`

Verified `CustomNotificationAdsStudy` is not listed

image


Verified System ads are shown:

image

**Review by kjozwiak — APPROVED** [review]
Uplift into `main/production` approved after deliberating with @brave/uplift-approvers. QA has verified the PR on both `1.72.x` & `1.71.x` as per https://github.com/brave/brave-variations/pull/1218#issuecomment-2394708661.

**Pull request merged**
